docs/components/modeler/web-modeler/git-sync.md

```diff
@@ -36,12 +36,16 @@ Click **Create GitHub App** to finish.
 3. Select **Only select repositories**, and choose the repository to sync with Web Modeler.
 4. Once redirected to your application's installation page, copy the **Installation ID** located at the end of the page's URL: `https://github.com/settings/installations/{installation_id}`.
 
-### Configure GitHub in Web Modeler
+### [Configure GitHub in Web Modeler](#configure-github-in-web-modeler)
 
 :::note
 An organization administration account (or project administrator in Camunda Self-Managed) is required for the initial GitHub configuration.
 :::
 
+:::note
+When using a self-hosted GitHub instance, ensure the environment variable `camunda.modeler.git-sync.github` is set to the API URL of your self-hosted GitHub instance. It usually looks like `http(s)://HOSTNAME/api/v3`. Refer to [GitHub documentation](https://docs.github.com/en/enterprise-server@3.15/rest/enterprise-admin?apiVersion=2022-11-28#endpoint-urls) and choose the correct enterprise server version.
```

Comment:
Probably, it would also make sense to document this on https://docs.camunda.io/docs/self-managed/modeler/web-modeler/configuration/#configuration-of-the-restapi-component, but this could also be done later (together with the GitLab base URL).